angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-portlet.html

```diff
@@ -1,5 +1,5 @@
 <div class="portlet-list-item" >
-  <h2>
+  <h2 id="appTitle">
```

Comment:
The id of a field must be unique across the whole document (the whole page). Read http://www.w3.org/TR/html4/struct/global.html#adef-id for more details.

I would suggest using a programmatic id, like `appTitle{{::portlet.title}}` or something like that.